packages/config/src/VitessceAutoConfig.js

```diff
@@ -13,21 +24,26 @@ class AbstractAutoConfig {
     throw new Error('The composeFileConfig() method has not been implemented.');
   }
 }
+
 class OmeTiffAutoConfig extends AbstractAutoConfig {
-  constructor(fileUrl) {
+  constructor(fileUrl, hintsConfig, hintsType, useHints) {
```

Comment:
If these extra parameters are only used in `composeViewsConfig`, can we move them as parameters to that function instead?

Likewise for the other classes that inherit from AbstractAutoConfig